script.js

```diff
@@ -0,0 +1,942 @@
+  }
+
+  const addRemoveKeyActive = (currentKey) => {
+    if (isKeydown) {
+      currentKey.classList.add(classes.KEY_ACTIVE);
+    } else {
+      currentKey.classList.remove(classes.KEY_ACTIVE);
+    }
+  };
+
+  let isRightKey = false;
+  let target;
+  for (let i = 0; i < elements.keys.length; i += 1) {
+    const key = elements.keys[i];
+    const keyText = key.innerText;
+    if ((keyText === variables.specialKeys.META && e.key === 'Meta')) {
+      addRemoveKeyActive(key);
+      if (isKeydown) {
+        key.firstElementChild.classList.remove(
+          isPlatformWindows() ? classes.META_WIN : classes.META_APPLE,
+        );
+        key.firstElementChild.classList.add(
+          isPlatformWindows() ? classes.META_WIN_ACTIVE : classes.META_APPLE_ACTIVE,
+        );
+      } else {
+        key.firstElementChild.classList.remove(
+          isPlatformWindows() ? classes.META_WIN_ACTIVE : classes.META_APPLE_ACTIVE,
+        );
+        key.firstElementChild.classList.add(
+          isPlatformWindows() ? classes.META_WIN : classes.META_APPLE,
+        );
+      }
+      break;
+    } else if (isSpecialKey(keyText, e)) {
+      addRemoveKeyActive(key);
+      target = key;
+      break;
+    } else if (isKeydown && !e.repeat
+      && (keyText === variables.specialKeys.CAPS_LOCK && e.key === 'CapsLock')) {
+      if (key.classList.contains(classes.KEY_ACTIVE)) {
+        key.classList.remove(classes.KEY_ACTIVE);
+      } else {
+        key.classList.add(classes.KEY_ACTIVE);
+      }
+      target = key;
+      break;
+    } else if (isKeydown && !e.repeat
+      && (keyText === variables.specialKeys.NUM_LOCK && e.key === 'NumLock')) {
+      if (key.classList.contains(classes.KEY_ACTIVE)) {
+        key.classList.remove(classes.KEY_ACTIVE);
+      } else {
+        key.classList.add(classes.KEY_ACTIVE);
+      }
+      target = key;
+      break;
+    } else if (isSecondKey(keyText, e)) {
+      if (isRightKey) {
+        addRemoveKeyActive(key);
+        target = key;
+        break;
+      }
+      isRightKey = true;
+    } else if (keyText === e.key) {
+      addRemoveKeyActive(key);
+      target = key;
+      break;
+    }
+  }
+
+  return target;
+};
+
+// ///////////////////////// KEYBOARD HANDLERS ///////////////////////////
+const handlerKeyDown = (e) => {
+  if (!e.repeat) {
+    playKeypressSound();
+
+    if (((e.ctrlKey && e.shiftKey) && (e.code === 'ControlLeft' || e.code === 'ShiftLeft'))
+    || ((e.ctrlKey && e.altKey) && (e.code === 'ControlLeft' || e.code === 'AltLeft'))
+    || ((e.shiftKey && e.altKey) && (e.code === 'ShiftLeft' || e.code === 'AltLeft'))) {
+      changeLanguage();
+    }
+  }
+
+  elements.textarea.focus();
+
+  const target = processKeySelection(e);
+  handlerKeyInput(target, e);
+};
+
+const handlerKeyUp = (e) => {
+  if (e.key === variables.specialKeys.SHIFT) {
+    variables.isShift = !variables.isShift;
+    changeCase();
+    changeOnShift();
+  }
+
+  const PRINT_SCREEN = 'PrintScreen';
```

Comment:
Declaration of the PRINT_SCREEN should be one the first line in function handlerKeyUp.

Reply:
I thought, it is always great to declarate variables as close as possible to the using